packages/integration-tests/src/helpers/admin-tenant.ts

```diff
@@ -25,7 +27,8 @@ export const createResponseWithCode = (statusCode: number) => ({
   response: { statusCode },
 });
 
-export const createUserWithAllRoles = async () => {
+// Assign all roles if `roleNames` not specified.
+const createUserWithSpecifiedRoleNames = async (roleNames?: string[]) => {
```

Comment:
misleading function names.  `createUserWithRolenames` roleNames should be required. 
Instead, query all roles in the `createUserWithAllRoles` method, and pass in all the roleNames to the method. 
Or maybe should give a better name.

Reply:
Refactored.